cupy/cuda/cudnn.pxd

```diff
@@ -133,7 +133,9 @@ cpdef destroyTensorDescriptor(size_t tensorDesc)
 cpdef addTensor_v2(
         size_t handle, int mode, size_t alpha, size_t biasDesc,
         size_t biasData, size_t beta, size_t srcDestDesc, size_t srcDestData)
-
+cpdef addTensor(
```

Comment:
The API of `addTensor` differs depending on the version of cuDNN. So use `addTensor_v2` or `addTensor_v3`. This fix is already done in the current master branch. So please merge or rebase the master branch to resolve this problem.